src/Service/ImageHandlerInterface.php

```diff
@@ -6,9 +6,45 @@
 
 interface ImageHandlerInterface
 {
+    /**
+     * Fetch image from remote server.
+     *
+     * @param string $url
+     *  Where to download the image from
+     *
+     * @return string
+     *   Relative path to the downloaded file
+     */
     public function fetch(string $url): string;
 
+    /**
+     * Remove image from the filesystem.
+     *
+     * @param Image $image
+     *   The image top remove
+     *
+     * @return bool
+     *   True if image was removed else false
+     */
     public function remove(Image $image): bool;
 
+    /**
+     * Make image transformations.
+     *
+     * @param image $image
+     *   The image to make derived/transformation from
+     */
     public function transform(Image $image): void;
+
+    /**
+     * Get urls for derived images from image url.
+     *
+     * @param string $imageUrl
+     *   The local url for the image
```

Comment:
Is “The local url for the image” the one returned by `fetch`? If so, rename to `$imagePath` and make a note on this.